examples/one-rate-plan/version.tf

```diff
@@ -6,7 +6,7 @@ terraform {
   required_providers {
     ibm = {
       source  = "ibm-cloud/ibm"
-      version = "1.56.1"
+      version = "1.60.0"
```

Comment:
As per the code comment above, this is purposely locked into 1.56.1, because the root module itself has a range constraint of `>= 1.56.1, < 2.0.0`, so we purposely lock into the lowest version in the range to ensure it remains working. Other examples are set up to use latest version of provider